### PR TITLE
Installation Tutorials

### PR DESCRIPTION
**Pull request by mwilman**
Reworked the README's.
What's your opinion on that?

### CODE REVIEW

**Comment by Wegmann-Brockhaus** [comment]
ok

**Comment by Wegmann-Brockhaus** [comment]
von meiner seite aus sieht das gut aus

**Review by KarasuRou — COMMENTED** [review]
Looks good to me.
But in the _tomcat_install.md_ i think the formation is a bit broken.
grafik

Also in _README.md_, i think the numbering should be different, or should the 9.1 come after the 10? ^^
grafik

**Comment by mwilman** [comment]
@KarasuRou oh shoot, you're right

File: docs/tomcat_install.md [hunk 1]
```diff
@@ -2,12 +2,15 @@
...
 1. Install and Setup a supported Database:
     - [PostgreSQL (Tutorial)](./postgres_install.md)
     - [Microsoft SQL Server (Tutorial)](./mssql_install.md)
+2. Install Java 8: https://www.java.com/en/download/ (**Only this version is supported**! Please make sure that Tomcat uses Java 8 JDK)
+3. Install Tomcat 9: https://tomcat.apache.org/download-90.cgi (Download the 32-bit/64-bit Windows Service Installer).<br>
+   *You also have to set up a gui account in the installation!*
 
-2. Install Java 8: https://www.java.com/en/download/ (Only this version is supported! Please make sure that Tomcat uses Java 8 JDK)
-3. Install Tomcat 9: https://tomcat.apache.org/download-90.cgi (Download the 32-bit/64-bit Windows Service Installer, you also have to set up a gui account in the installation!)
+####Deploying Release
```
Comment (KarasuRou) [inline docs/tomcat_install.md:13]:
```suggestion
#### Deploying Release
```

Perhapse try the suggestion above?
I still have the buggy formation.

grafik

**Review by KarasuRou — CHANGES_REQUESTED** [review]
Still the same formatation in _tomcat_install.md_.
Perhapse try a space after "####"?

**Review by KarasuRou — APPROVED** [review]

**Pull request merged**